awx/main/tasks.py

```diff
@@ -2046,35 +2046,24 @@ def build_env(self, project_update, private_data_dir, isolated=False, private_da
         env['ANSIBLE_CALLBACK_PLUGINS'] = self.get_path_to('..', 'plugins', 'callback')
         if settings.GALAXY_IGNORE_CERTS:
             env['ANSIBLE_GALAXY_IGNORE'] = True
-        # Set up the public Galaxy server, if enabled
-        galaxy_configured = False
-        if settings.PUBLIC_GALAXY_ENABLED:
-            galaxy_servers = [settings.PUBLIC_GALAXY_SERVER]  # static setting
-        else:
-            galaxy_configured = True
-            galaxy_servers = []
-        # Set up fallback Galaxy servers, if configured
-        if settings.FALLBACK_GALAXY_SERVERS:
-            galaxy_configured = True
-            galaxy_servers = settings.FALLBACK_GALAXY_SERVERS + galaxy_servers
-        # Set up the primary Galaxy server, if configured
-        if settings.PRIMARY_GALAXY_URL:
-            galaxy_configured = True
-            galaxy_servers = [{'id': 'primary_galaxy'}] + galaxy_servers
-            for key in GALAXY_SERVER_FIELDS:
-                value = getattr(settings, 'PRIMARY_GALAXY_{}'.format(key.upper()))
-                if value:
-                    galaxy_servers[0][key] = value
-        if galaxy_configured:
-            for server in galaxy_servers:
-                for key in GALAXY_SERVER_FIELDS:
-                    if not server.get(key):
-                        continue
-                    env_key = ('ANSIBLE_GALAXY_SERVER_{}_{}'.format(server.get('id', 'unnamed'), key)).upper()
-                    env[env_key] = server[key]
-            if galaxy_servers:
-                # now set the precedence of galaxy servers
-                env['ANSIBLE_GALAXY_SERVER_LIST'] = ','.join([server.get('id', 'unnamed') for server in galaxy_servers])
+
+        # build out env vars for Galaxy credentials (in order)
+        galaxy_server_list = []
```

Comment:
@wenottingham @rebeccahhh @chrismeyersfsu @AlanCoding one behavioral change worth pointing out here is that *for new installs* and *new Organizations* (ignore old settings that get migrated on upgrade), the default behavior is "don't pull from galaxy.ansible.com by default".  I _think_ this is what I heard @wenottingham say he wanted - is that right?

Reply:
We'd need to document this, but I'm OK with it

Reply:
duly noted :+1:

Reply:
So I think this is the API behavior we landed on, but I think as a nicety in the UI, when you create a new org, we're talking about pre-filling the field to include Ansible Galaxy.  So at the time users create an org, they are immediately given the choice to opt in/out of "community Galaxy" at org creation time.

Reply:
see: https://github.com/ansible/awx/pull/7817#discussion_r465950688